src/main/java/org/broadinstitute/consent/http/db/mapper/DatasetReducer.java

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.consent.http.db.mapper;
+
+import org.broadinstitute.consent.http.models.DataSet;
+import org.broadinstitute.consent.http.models.DataUse;
+
+import org.jdbi.v3.core.result.LinkedHashMapRowReducer;
+import org.jdbi.v3.core.result.RowView;
+import java.util.Objects;
+
+import java.util.Map;
+
+public class DatasetReducer implements LinkedHashMapRowReducer<Integer, DataSet> {
+  @Override
+  public void accumulate(Map<Integer, DataSet> map, RowView rowView) {
+    DataSet dataset = map.computeIfAbsent(
+      rowView.getColumn("datasetid", Integer.class),
+      id -> rowView.getRow(DataSet.class));
+    
+      if(Objects.nonNull(rowView.getColumn("dataUse", String.class))) {
+        dataset.setDataUse(
+          DataUse.parseDataUse(
```

Comment:
Not a request for a change, just some background. The parse method here is rather old code that uses Jackson. I've come to the conclusion that Gson is a much better fit for this kind of task.